python/cugraph/cugraph/dask/community/egonet.py

```diff
@@ -110,6 +111,15 @@ def ego_graph(input_graph, n, radius=1, center=True):
     # Initialize dask client
     client = input_graph._client
 
+    # FIXME: Implement a better way to check if the graph is weighted similar
+    # to 'simpleGraph'
+    if len(input_graph.edgelist.edgelist_df.columns) != 3:
```

Comment:
Is `input_graph` a cugraph `Graph` instance?  If so, you should be able to just call [input_graph.is_weighted()](https://github.com/rapidsai/cugraph/blob/a189146a4e0c946a497bd9794ac88b881e31c9d7/python/cugraph/cugraph/structure/graph_classes.py#L578).  Although, this should just be removed if there's a problem in the C API that gets fixed based on @ChuckHastings comment.

Also, the docstring needs updating (I can't add a comment to that in this PR) - it still uses "G" instead of "input_graph", and it also mentioned weights are allowed which should be removed if we decide not to allow them.

Reply:
The choice - in 23.02 (once I fix the C API) - will be as follows:
* Is it better to check `input_graph.is_weighted()` and avoid the call, or
* Is it better to call the function to get the weights and check if you got a null pointer back

Reply:
> and it also mentioned weights are allowed which should be removed if we decide not to allow them
`egonet` supports weighted graphs